### PR TITLE
atd and atdgen

### PR DESCRIPTION
**Pull request by pirbo**
I've kept the `install` files as if because it is what you do in yojson and biniou but I'm not sure I'm right...

### CODE REVIEW

**Comment by whitequark** [comment]
Why are you packaging atdgen? If I remember correctly, atdgen does not have any libraries that should run on the host system (Windows). The policy of opam-cross-* repositories (so far) is to only package libraries (mostly because of trouble with distributing supplementary data).

**Comment by pirbo** [comment]
Sadly no.
There is an atdgen.cmxa that contains (at least) the modules Ag_util and Ag_oj_run that generated files intensively use.

File: packages/atd-windows.1.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+maintainer: "pierre.boutillier@laposte.net"
+authors: "Martin Jambon"
+homepage: "https://github.com/mjambon/atd"
+bug-reports: "https://github.com/mjambon/atd/issues"
+dev-repo: "https://github.com/mjambon/atd.git"
+license: "BSD3"
+patches: ["patches/ocamlc_config_through_ocamlfind.patch"]
+build: ["env" "OCAMLFIND_TOOLCHAIN=windows" make]
+install:
+["env" "OCAMLFIND_TOOLCHAIN=windows" make "install" "BINDIR=%{bin}%"]
```
Comment (whitequark) [inline packages/atd-windows.1.2.0/opam]: this *definitely* shouldn't be `%{bin}%`, rather it should be within the ocaml-windows root

File: packages/atdgen-windows.1.10.0/files/atdgen.install [hunk 2]
```diff
@@ -0,0 +1,6 @@
+bin: [
+  "src/atdgen.run" {"atdgen.run"}
+  "src/atdgen" {"atdgen"}
+  "atdgen-cppo/atdgen-cppo"
+  "atdgen-cppo/cppo-json"
+]
```
Comment (whitequark) [inline packages/atdgen-windows.1.10.0/files/atdgen.install]: Ditto here--if you install these at all, install them into the cross-root, or they will clobber the files installed by `atdgen` proper in the prefix, and likely breaking your build scripts.

Comment (pirbo) [inline packages/atdgen-windows.1.10.0/files/atdgen.install]: I fully agree and I was about to remove the `atdgen.install` file when I realized that you kept `/packages/yojson-windows.1.3.2/files/yojson.install`. 
I was then confused but I leave there both `atdgen.install` and `atd.install`!

File: packages/yojson-windows.1.3.2/files/yojson.install [hunk 3]
```diff
@@ -1 +0,0 @@
-bin: ["ydump"]
```
Comment (whitequark) [inline packages/yojson-windows.1.3.2/files/yojson.install]: Good catch--I'm not sure what was I thinking.

**Pull request merged**